libraries/standard.js

```diff
@@ -1,8 +1,32 @@
-const lib = new Map();
```

Comment:
this should be added back in as we're returning a map of functions and not just the CSV parser